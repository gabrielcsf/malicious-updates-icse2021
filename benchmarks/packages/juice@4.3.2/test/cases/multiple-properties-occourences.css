.a {
  color: #800000;
  color: red;
  color: rgba(255,0,0,.5);
  color: -prefix-unknownvalue;
}
.b {
  color: blue;
}
.b {
  color: rgba(0,0,255,.5);
}
.c {
  color: red !important;
  color: rgba(255,0,0,.5) !important;
  color: blue;
}
.d {
  padding: 5px;
  padding-top: 20px;
  padding: 10px;
  padding-top: unexpected value;
  padding: unknown;
  padding-top: again something unexpected;
}
