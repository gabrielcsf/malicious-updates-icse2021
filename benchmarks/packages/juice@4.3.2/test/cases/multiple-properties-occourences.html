<div class="a"></div>
<div class="b"></div>
<div class="c"></div>
<div class="d"></div>
