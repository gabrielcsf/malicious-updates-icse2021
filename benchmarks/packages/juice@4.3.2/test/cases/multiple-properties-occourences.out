<div class="a" style="color: #800000; color: red; color: rgba(255,0,0,.5); color: -prefix-unknownvalue;"></div>
<div class="b" style="color: rgba(0,0,255,.5);"></div>
<div class="c" style="color: red; color: rgba(255,0,0,.5);"></div>
<div class="d" style="padding: 5px; padding-top: 20px; padding: 10px; padding-top: unexpected value; padding: unknown; padding-top: again something unexpected;"></div>
